frontend/src/components/FilterDropdown.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {
+    Box,
+    Button,
+    Menu,
+    MenuButton,
+    MenuList,
+    MenuItemOption,
+    MenuOptionGroup,
+} from '@chakra-ui/react'
+import { ChevronDownIcon } from '@chakra-ui/icons'
+
+const FilterDropdown = (props) => {
+    const { filter, setFilter } = props
+
+    const TICKET_TYPES = Object.freeze({
```

Comment:
Great thinking, but we actually have something called DATA_KEYS on dashboard.js. If you can rename that to TICKET_TYPES and export it instead, you can import it in this file so we avoid redeclaring this